web/packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -314,6 +314,11 @@ const EventIconMap: Record<EventCode, any> = {
   [eventCodes.AUTOUPDATE_AGENT_ROLLOUT_TRIGGER]: Icons.RocketLaunch,
   [eventCodes.AUTOUPDATE_AGENT_ROLLOUT_FORCE_DONE]: Icons.Checks,
   [eventCodes.AUTOUPDATE_AGENT_ROLLOUT_ROLLBACK]: Icons.Restore,
+  [eventCodes.MCP_SESSION_START]: Icons.Info,
+  [eventCodes.MCP_SESSION_END]: Icons.Info,
+  [eventCodes.MCP_SESSION_REQUEST]: Icons.Info,
+  [eventCodes.MCP_SESSION_REQUEST_FAILURE]: Icons.Warning,
+  [eventCodes.MCP_SESSION_NOTIFICATION]: Icons.Info,
```

Comment:
We should find a better icon. Maybe a brain or something?

@roraback any ideas for an audit log icon for MCP?

Reply:
maybe this one?
https://github.com/modelcontextprotocol/modelcontextprotocol/blob/main/docs/favicon.svg

Reply:
updated

slack thread for reference
https://gravitational.slack.com/archives/C02DQ1C2BMW/p1747945158708939